hack/update-staging-godeps-dockerized.sh

```diff
@@ -87,6 +87,15 @@ function diffGodepManifest() {
   fi
 }
 
+# Create a fake git repo for staging to prevent godeps from complaining
+pushd "${KUBE_ROOT}/staging/src" >/dev/null
```

Comment:
Is there a reason you're doing this for `${KUBE_ROOT}/staging/src` as opposed to just doing it in `${KUBE_ROOT}`?